### PR TITLE
Added 10 jokes to jokes.json

### PR DESCRIPTION
**Pull request by karmanya007**
Sorry, the prettier extension spaced out the code, so all of jokes.json shows up as new. I have add jokes from 142-152.

### CODE REVIEW

File: src/jokes.json [hunk 1]
```diff
@@ -1,644 +1,683 @@
...
+		"a": "Try catch.",
+		"form": "qa"
+	},
+	"142": {
+		"q": "Why arrays Chuck Norris declares are of infinite size.",
+		"a": "Because Chuck Norris knows no bounds.",
+		"form": "qa"
+	},
+	"143": {
+		"q": "Why doesm't Chuck Norris doesn’t need garbage collection.",
```
Comment (ABSphreak) [inline src/jokes.json:650]: Fix grammar and spelling mistakes.

File: src/jokes.json [hunk 2]
```diff
@@ -1,644 +1,683 @@
...
+	},
+	"148": {
+		"q": "Why did the geek add body { padding-top: 1000px; } to his Facebook profile?",
+		"a": "He wanted to keep a low profile.",
+		"form": "qa"
+	},
+	"149": "8 bytes walk into a bar, the bartenders asks \"What will it be?\" One of them says, \"Make us a double.\"",
+	"150": "An SEO couple had twins. For the first time, they were happy with duplicate content.",
+	"151": "LISP - Lots of Infuriating Superfluous Parentheses",
+	"152": {
```
Comment (ABSphreak) [inline src/jokes.json:678]: Make this one a multi-line one, not a Q/A type.

**Review by ABSphreak — CHANGES_REQUESTED** [review]

File: src/jokes.json (hunk 2 above)
Comment (ABSphreak) [reply src/jokes.json:678]: Number 152

Comment (karmanya007) [reply src/jokes.json:678]: okay

**Review by ABSphreak — APPROVED** [review]

**Comment by karmanya007** [comment]
Could you please tag this PR with "hacktoberfest-accepted" as as it is showing up as an ineligible PR.

**Comment by ABSphreak** [comment]
There you go.

**Comment by karmanya007** [comment]
Sure 😃

**Pull request merged**